### PR TITLE
I created the Department, Employee and Supervisor classes

### PR DESCRIPTION
**Pull request by ziadnasser19**
I suposed edit and super view methods in command are changed

### CODE REVIEW

**Review by mistaluai — APPROVED** [review]
Neat work!

File: src/Backend/Entities/Employee.java [hunk 1]
```diff
@@ -1,7 +1,95 @@
...
+    private final String name;
+    private final String position;
+    private final int id;
+    private final Department department;
+    private final AttendanceManager attendanceManager;
+
+    public Employee(int id) {
+        this.id = id;
+        attendanceManager = new AttendanceManager();
+        isSuperVisor();
```
Comment (BlindFl0W) [inline src/Backend/Entities/Employee.java:24]: This doesn't need to be here. Remove it.

File: src/Backend/Entities/Employee.java [hunk 2]
```diff
@@ -1,7 +1,95 @@
...
+        IAttendanceDataHandler iAttendanceDataHandler = new AttendanceFileHandler();
```
Comment (BlindFl0W) [inline src/Backend/Entities/Employee.java:25]: This constractor should take the ID. needs to be edited after the attendance manager edit.

File: src/Backend/Entities/Supervisor.java
Comment (BlindFl0W) [inline src/Backend/Entities/Supervisor.java:1]: Looks Good.

**Pull request merged**